packages/template-retail-react-app/app/ssr.js

```diff
@@ -29,7 +29,14 @@ const options = {
     // The protocol on which the development Express app listens.
     // Note that http://localhost is treated as a secure context for development,
     // except by Safari.
-    protocol: 'http'
+    protocol: 'http',
+
+    // Option for whether to set up a special endpoint for handling
+    // private SLAS clients
+    // Set this to false if using a SLAS public client
+    // When setting this to true, make sure to also set the SLAS_PRIVATE_CLIENT_SECRET
+    // environment variable as this endpoint will return HTTP 501 if it is not set
+    useSLASPrivateClient: false
```

Comment:
I see we're not defaulting this to on _yet_ – but it is interesting to think about when this is defaulted to on, how do we want folks to provide `SLAS_PRIVATE_CLIENT_SECRET`?

Do we always expect them to pass in an environment variable for `SLAS_PRIVATE_CLIENT_SECRET`, likely changing their `npm start` command? Or would we like to add support for `.env` files or something like that?

Reply:
One thing we will want to consider if we do eventually add `.env` files is that we may want to keep the client secret outside of the repository so that it is not accidentally shared with other users.